lib/protocol/binary.ex

```diff
@@ -1,5 +1,5 @@
 defmodule Hippy.Protocol.Binary do
   defmacro byte do
-    quote do: signed-8
```

Comment:
Normally the version without spaces is more common in typedefs

Reply:
@mpichette the formatter is changing this but I also noticed this isn't even being used, I'm also not sure if this is doing what I think it's suppose to. I think Elixir is interpreting this as the variable `signed` `minus` `8`, is that the intention?

Reply:
Should I just delete this module maybe?